src/robot/libdocpkg/htmlutils.py

```diff
@@ -1,5 +1,4 @@
-#  Copyright 2008-2015 Nokia Networks
-#  Copyright 2016-     Robot Framework Foundation
+#  Copyright 2020-     Robot Framework Foundation
```

Comment:
What's this change? Is this a new file where you don't want have old Nokia copyrights? I think it would be better to leave them because I think the file anyway contains old code that has been moved. It's also easier if all files have same copyrights.

Reply:
can do that.

was just not there before.
But no opinion on that.